Gemfile

```diff
@@ -15,14 +15,15 @@ gem 'image_processing', '1.9.3'
 gem 'mini_magick', '4.9.5'
 gem 'active_storage_validations', '0.8.2'
 gem "aws-sdk-s3", require: false
+gem 'pry-rails'
 
 group :development, :test do
   gem 'sqlite3', '~> 1.4'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
 end
 
 group :production do
-  gem 'pg', '0.20.0'
+  gem 'pg'
```

Comment:
I would add the version back unless there was a reason it was removed

Reply:
nope, I will